pkg/network/setup/netpod/netpod.go

```diff
@@ -156,7 +156,7 @@ func (n NetPod) Setup() error {
 		if err != nil {
 			return err
 		}
-		log.Log.Infof("Current pod network: %s", currentStatusBytes)
+		log.Log.With("uid", n.vmiUID).Infof("Current pod network: %s", currentStatusBytes)
```

Comment:
I think this is great improvement but I generally like the following
```
func (l FilteredLogger) Object(obj LoggableObject) *FilteredLogger {

	name := obj.GetObjectMeta().GetName()
	namespace := obj.GetObjectMeta().GetNamespace()
	uid := obj.GetObjectMeta().GetUID()
	kind := obj.GetObjectKind().GroupVersionKind().Kind

	logParams := make([]interface{}, 0)
	if namespace != "" {
		logParams = append(logParams, "namespace", namespace)
	}
	logParams = append(logParams, "name", name)
	logParams = append(logParams, "kind", kind)
	logParams = append(logParams, "uid", uid)

	l.with(logParams...)
	return &l
}
```

This allow us to both search by name and uid. Both of which are important as the name could be reused... The problem of having sometimes one or another or both is that you need to search twice. One for name and one for uid. This makes life of the person debugging a problem little bit more hard. 

Therefore maybe we want to pass the name as well or just create a good logger when we create  `NetPod`. WDYT?

Reply:
I will try to create a logger as part of the NetPod instantiation.

Generally, the UID is the must, all others are optional. But it is very common to add everything, and being consistent is important.

Reply:
Done